CodeTracker/CodeTrackerUnitTest/CodingTrackerInputTest.cs

```diff
@@ -0,0 +1,102 @@
+using CodeTracker;
+using Spectre.Console.Testing;
+namespace CodeTrackerUnitTest;
+
+[TestClass]
+public class CodingTrackerInputTest
+{
+    private readonly TestConsole _console;
+    private readonly Input _input;
+    public CodingTrackerInputTest()
+    {
+        _console = new TestConsole();
+        _input = new Input(_console);
+    }
+    [TestMethod]
+    [DataRow(" ")]
+    [DataRow("description")]
+    public void TestingDescriptionInput(string input)
```

Comment:
🟠 This test checks for whitespace handling but uses `Assert.ThrowsException<InvalidOperationException>()` for invalid input.

💡 Use `Assert.ThrowsException<ArgumentException>()` instead if the method is expected to reject whitespace-only input. This aligns with .NET's standard for invalid arguments.

🔧 Split the test into two separate methods:
- One for valid input.
- One for invalid (whitespace) input.